### DISCUSSION TITLE
[discuss] Allow names to be unicode rather than ascii

### DISCUSSION THREAD

**Discussion by rufuspollock** [frictionlessdata/frictionlessdata.io#647] [Ideas]
In v1.0 we restricted names to be ascii lower case primarily to support reuse in urls without escaping.

However, this is a big restriction for users with non-english alphabet who may want to name their resources and packages using their own alphabets.

Questions

* What is status of IRI type stuff in browsers and elsewhere?
* How much of a problem is this for users right now?
* What serialization of unicode do we support (utf8 probably ...)?

**Comment by augusto-herrmann** [comment]
Good question, @rufuspollock.

I think IRI have been well supported in browsers for years. [According to Wikipedia](https://en.wikipedia.org/wiki/Internationalized_Resource_Identifier#Compatibility), UTF-8 encoding should be used for IRIs.

I'm all for it.
